sql/core/src/test/resources/sql-tests/inputs/regexp-function.sql

```diff
@@ -0,0 +1,15 @@
+SELECT regexp_extract('1a 2b 14m', '\\d+');
+
+SELECT regexp_extract('1a 2b 14m', '\\d+', 0);
+
+SELECT regexp_extract('1a 2b 14m', '\\d+', 1);
+
+SELECT regexp_extract('1a 2b 14m', '\\d+', 2);
+
+SELECT regexp_extract('1a 2b 14m', '(\\d+)([a-z]+)');
+
+SELECT regexp_extract('1a 2b 14m', '(\\d+)([a-z]+)', 0); 
+
+SELECT regexp_extract('1a 2b 14m', '(\\d+)([a-z]+)', 1); 
+
+SELECT regexp_extract('1a 2b 14m', '(\\d+)([a-z]+)', 2);
```

Comment:
Can you move these tests into `sql-tests/inputs/string-functions.sql` insteadf of making a new file?

Reply:
Also, can you add tests in `RegexpExpressionsSuite`, too , for the non-codegen path?

Reply:
OK